DEV_GUIDELINES.md

```diff
@@ -0,0 +1,72 @@
+# Polywrap Core Development Guidelines
+
+Guidelines for the core development contributors on the Polywrap monorepo written to have a better understanding of Polywrap progress and coordination. Recommended read for new developers as part of onboarding. 
+
+## Issues management
+
+All planned features, bugs and enhancements are represented as a Github issue with an appropriate description, examples and issue labels.
+
+Issues should be part of an according Github project that has an automated kanban board consisting of columns that mark the issue status and can be *Unassigned*, *Assigned*, *In progress* and *Done*. *Backlog* column is used for keeping the available issues that are up for grabs and also when creating new tasks such as features, bug fixes or ideas. 
+
+![Github project board](https://i.imgur.com/aLWa5HQ.png)
+
+Github projects are created for each project where there is always one active project board for core development i.e. [Prealpha board](https://github.com/polywrap/monorepo/projects/1) and others for development of other parts like Polywrappers such as [Uniswap](https://github.com/polywrap/monorepo/projects/2).
```

Comment:
Since we have moved integrations in the separate https://github.com/polywrap/integrations repo we can have project boards for integrations there.